docs/example_system.rst

```diff
@@ -0,0 +1,30 @@
+Example system
+===============
+
+Components in dashed boxes are drawn by hand using, e.g.,
+`Avogadro <http://avogadro.cc>`_ or generated elsewhere. Each
+component is wrapped as a simple python class with user defined attachment
```

Comment:
This is probably not the place for this edit, but just putting here at the moment.

Im not a big fan of using words like "simple" when trying to introduce users, as this can lead to negative connotations when they run into issues.

```suggestion
component is wrapped as a python class with user defined attachment
```